### ISSUE TITLE
is there any way to make any other file type of instead if an exe?

### ISSUE THREAD

**Issue by aayush66991** [UnamSanctam/SilentCryptoMiner#1970]
is there any way to make any other file type of instead if an exe and it would be a great option to select extension while building file.

**Comment by UnamSanctam** [comment]
No, since then you wouldn't be able to run it. Though you can use an extension spoofer (there are many online) if you want to spoof the extension though extension spoofing isn't perfect of course.

**Comment by alpacinoGits** [comment]
You can build droppers that you can randomize method and function names daily, weekly or monthly depending on your spread rate. You can use extensions such as py, bat, ps1 and more with this way. You can also research extension exploits such as lnk.

**Comment by aayush66991** [comment]
Yeah, you are right but what if I don't wanna send an exe any other
extension I can use?
Thanks

**Comment by UnamSanctam** [comment]
>Yeah, you are right but what if I don't wanna send an exe any other
extension I can use?
Thanks

Either you can use an extension spoofer that with some limitations can make the extension look like something else. Or you can do as @alpacinoGits said and use some dropper/binder/downloader (or special crypters) that output their scripts as .py, .bat, .psi1 or other formats.
